homework/01LessonOne.js

```diff
@@ -204,11 +207,11 @@ var stringFourStrictNotEqualsNumberFour;
 //
 // Your Answer Goes Here:
 //
-//
+// == is equal to so '8' == 8 is true because it is the same # even though one is a string. === is equal to the same value and type '8'=== 8 is false because they are not the same value. '8' is a string not a variable and 8 would be a variable.
 // What’s the difference between != and !==?
```

Comment:
So in the first statement i said it correct  == is equal to so '8' == 8 is true because it is the same # even though one is a string. === is equal to the same value and type
 …i just mixed up the terms when i was trying to show it out. 

 

> On Jul 31, 2016, at 3:18 PM, Piero A. Pretto <notifications@github.com> wrote:
> 
> In homework/01LessonOne.js <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/55#discussion_r72910053>:
> 
> > @@ -204,11 +207,11 @@ var stringFourStrictNotEqualsNumberFour;
> >  //
> >  // Your Answer Goes Here:
> >  //
> > -//
> > +// == is equal to so '8' == 8 is true because it is the same # even though one is a string. === is equal to the same value and type '8'=== 8 is false because they are not the same value. '8' is a string not a variable and 8 would be a variable.
> >  // What’s the difference between != and !==?
> "8" and 8 have the same value. "===" checks to equality in value AND data type.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/55/files/6a3a2039b6d1be8f3c24c2d6f6a75a5a18583e74..b855664e4a98a72bd32d06dd67d14a9b6b98441b#r72910053>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ASRVxQnX2wS4YGMFti19DEaeU45UZ7q_ks5qbQMHgaJpZM4JUsvF>.
>